riptide-stream/src/test/java/org/zalando/riptide/stream/StreamConverterTest.java

```diff
@@ -47,30 +48,22 @@
 
     @Test
     public void shouldSupportMediaTypes() throws Exception {
-        final List<MediaType> medias = new StreamConverter<>().getSupportedMediaTypes();
+        final List<MediaType> medias = streamConverter().getSupportedMediaTypes();
         assertThat(medias, hasItem(APPLICATION_X_JSON_STREAM));
         assertThat(medias, hasItem(APPLICATION_JSON_SEQ));
     }
 
     @Test
     public void shouldSupportRead() throws Exception {
-        final ObjectMapper mapper = mock(ObjectMapper.class);
-        final TypeFactory factory = new ObjectMapper().getTypeFactory();
-        when(mapper.getTypeFactory()).thenReturn(factory);
-        when(mapper.canDeserialize(any())).thenReturn(true);
-        final StreamConverter<?> unit = new StreamConverter<>(mapper, singletonList(APPLICATION_JSON));
+        final HttpMessageConverter<Stream<Object>> unit = streamConverter(new ObjectMapper(), singletonList(APPLICATION_JSON));
 
         assertFalse(unit.canRead(Object.class, APPLICATION_XML));
         assertFalse(unit.canRead(Stream.class, APPLICATION_JSON));
-
-        assertTrue(unit.canRead(Object.class, APPLICATION_JSON));
-        assertTrue(unit.canRead(List.class, APPLICATION_JSON));
-        assertTrue(unit.canRead(List[].class, APPLICATION_JSON));
-        assertTrue(unit.canRead(AccountBody.class, null));
-        assertTrue(unit.canRead(AccountBody[].class, null));
-
-        when(mapper.canDeserialize(factory.constructType(AccountBody.class))).thenReturn(false);
-        assertFalse(unit.canRead(AccountBody.class, APPLICATION_JSON));
+        assertFalse(unit.canRead(Object.class, APPLICATION_JSON));
+        assertFalse(unit.canRead(List.class, APPLICATION_JSON));
```

Comment:
It will not allow to read

```json
["a"]
["b"]
["c"]
```
into a `Stream<List<String>>`.

It will **not** allow to read:

```json
[
  "a",
  "b",
  "c"
]
```
into a `Stream<String>`.